### ISSUE TITLE
Invalid svg in CoachmarkBeacon component scss

### ISSUE THREAD

**Issue by jdelattr** [carbon-design-system/ibm-products#3728]
## What package(s) are you using?

- [x] Carbon for IBM Products (or Carbon for IBM Cloud & Cognitive)
- [ ] Legacy/CDAI
- [ ] Legacy/Security

## Detailed description

> Describe in detail the issue you're having.

The 2 svg defined [here](https://github.com/carbon-design-system/ibm-products/blob/32584f94f46ab998f6d47c5b98e716018a454c53/packages/ibm-products/src/components/CoachmarkBeacon/_coachmark-beacon.scss#L51-L52) are invalid and are breaking application build using create-react-app (react-scripts 5.0.1)

> Is this issue related to a specific component?

CoachmarkBeacon

> What did you expect to happen? What happened instead? What would you like to
> see changed?

Update the svg to prevent postcss-svgo to error (missing space?)

> What browser are you working in?

No browser involved

> What version of the @carbon/ibm-products (or @carbon/ibm-cloud-cognitive)
> package are you using?

@carbon/ibm-products 1.61.0

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?

As soon as possible unfortunately 😅

## Steps to reproduce the issue

1. Try to build an app with CRA using @carbon/ibm-products 1.61.0

> Please create a reduced test case in CodeSandbox
>
> https://codesandbox.io/s/github/carbon-design-system/ibm-products/tree/main/examples/carbon-for-ibm-products/SidePanel

## Additional information

- Screenshots or code

<img width="785" alt="Screenshot 2023-10-31 at 15 11 53" src="https://github.com/carbon-design-system/ibm-products/assets/7858897/9d1d0228-a3d7-492e-802e-2337885ba435">

```
postcss-svgo: /Users/jerome/git/somepath/static/css/main.dd2ec503.css:9:383740: Error: SvgoParserError: <input>:1:108: Invalid character in tag name
[INFO] 
[INFO] > 1 | …width='80px' height='80px' viewBox='008080'><pathfill='white' stroke='none…
```

- Notes

**Comment by paul-balchin-ibm** [comment]
Hi @jdelattr,

I've created a pull request to fix this issue. I've removed the entire CSS class this issue is associated with as it seems to have been part of an abandoned feature.

I can't replicate the error you seem to be having with the codesandbox URL you gave above.

You might be able to see if the fix works for you by going into your local app:
1. Open this file `./node_modules/@carbon/ibm-products/src/components/CoachmarkBeacon/_coachmark-beacon.scss`
2. Delete the `&-ring` class entirely
3. Save the file.
4. Restart your dev server.
5. See if that fixes the problem.
